app/src/main/admin/components/Touchstones/Coverage/UploadCoverage.tsx

```diff
@@ -25,20 +26,49 @@ class UploadCoverageComponent extends React.Component<UploadCoverageProps, Uploa
     render(): JSX.Element {
         return <form encType="multipart/form-data"
                      onSubmit={this.onSubmit}
-                     onChange={this.onChange}
                      noValidate>
-            <CustomFileInput required={true} accept=".csv" key={this.state.fileInputKey.toISOString()}>
+            <div className="form-group">
+                <label>Please provide any additional details on this coverage data such as:
+                    <ul>
+                        <li>
+                            caveats
+                        </li>
+                        <li>
+                            variable interpretations
+                        </li>
+                        <li>
+                            data sources
+                        </li>
+                        <li>
+                            things to note
+                        </li>
+                        <li>
+                            anything that could affect usage
+                        </li>
+                    </ul>
+                    Please see <InternalLink href={"coverage-variables"}>here</InternalLink> for information about how
```

Comment:
I think there's something up with this link / the breadcrumbs. If you click back and forth a bit, this link seems to get broken (points to touchstone/coverage/coverage-variables instead of touchstone/coverage-variables) and the same if you click on the breadcrumb link in the variables page itself.

Reply:
Ah, my bad, mismatch between breadcrumbs and router/link. It should all be `coverage/coverage-variables`, will update